src/main/webapp/i18n/de/submission.json

```diff
@@ -20,7 +20,8 @@
             "type": "Type",
             "result": "Ergebnis",
             "participation": "Participation",
-            "lockedSubmissionsLimitReached": "Du hast die maximale Anzahl von unvollständigen Bewertungen in diesem Kurs erreicht. Bitte beende deine offenen Bewertungen, bevor du eine neue beginnen kannst."
+            "lockedSubmissionsLimitReached": "Du hast die maximale Anzahl von unvollständigen Bewertungen in diesem Kurs erreicht. Bitte beende deine offenen Bewertungen, bevor du eine neue beginnen kannst.",
+            "resultTimeout": "Das Ergebnis zu deiner Abgabe mit der ID {{ id }} ist nicht eingetroffen. Bitte gebe deinen Code erneut ab."
```

Comment:
ID is not good here, because we should not show IDs to the student. I guess we can leave it out and simply say "deiner letzten Abgabe"

Reply:
```suggestion
            "resultTimeout": "Das Ergebnis zu deiner letzten Abgabe ist nicht verfügbar. Bitte gebe deinen Code erneut ab."
```